validator/laichikok-upgrade/README.md

```diff
@@ -0,0 +1,198 @@
+# LaiChiKok Upgrade (v2.0.0)
+
+LaiChiKok is the second major upgrade to the LikeCoin chain.
+
+## Changelog
+
+- Support new `like` account prefix. The legacy `cosmos` prefix is still supported.
+- Update Cosmos SDK to 0.44.6, ibc-go to 2.1.0.
+
+## Timeline
+
+|Event           |Time                     |
+|----------------|-------------------------|
+|Testnet Proposal|W1 April 2022            |
+|Testnet Deploy  |W2 April 2022, height TBC|
+|Mainnet Proposal|W2 April 2022            |
+|Mainnet Deploy  |W3 April 2022, height TBC|
+
+## Software Versioning Changes
+
+After `fotan-1.1`, releases will be tagged with semantic versioning to facilitate the usage of CI/CD pipelines, e.g. `v1.2.3`. Pre-release versions will contain suffixes such as `-rc1`, `-beta2`, `-alpha3`.
+
+Below is a list of recent stable releases:
+
+- `fotan-1.1`: Current mainnet version
+- `v1.2.0`: First version to support cosmovisor, binary-compatible with mainnet
+- `v2.0.0`: LaiChiKok upgrade
+
+## System Requirements During Upgrade
+
+The system resources required during the upgrade will be higher than normal operation for data migration tasks.
+
+In particular, around 12-20GB memory will be used, and data files will be duplicated for backup.
+
+After the upgrade is applied, the resources requirement is same as previously suggested in our [Node Setup Guide](../likecoin-chain-node/setup-a-node.md).
+
+### Recommended
+
+- 4 CPU Cores
+- 32GB RAM
+- 500GB+ SSD
```

Comment:
The recommendation is based on a test run on recent test run of migration with mainnet data. Any machine with less than 16GB will likely result in OOM(Out of memory) error.

Reply:
added